modules/UpgradeWizard/language/en_us.lang.php

```diff
@@ -282,7 +282,8 @@
 	//adding the string used in tracker. copying from homepage
 	'LBL_HOME_PAGE_4_NAME' => 'Tracker',
 	'LBL_CURRENT_PHP_VERSION' => '(Your current php version is ',
```

Comment:
To help translators its better to not have Round brackets or parentheses open and closing ( ) in different strings. It could make sentences strange in some language. Same goes for the space and full stop that starts the strings 'LBL_RECOMMENDED_PHP_VERSION_1' ` . Recommended php version is `
So maybe removing the  open and closing Round brackets from the strings and then make sure space and full stop are in used to end a string not to start it.